### PR TITLE
Created Logo API for Team logo images

### PR DESCRIPTION
**Pull request by ryanbarlow1**

### CODE REVIEW

File: backend/internal/server/apis/groups/logo.go
Comment (Skylude) [inline backend/internal/server/apis/groups/logo.go:30]: 👍🏿

File: backend/internal/server/apis/groups/logo.go
Comment (Skylude) [inline backend/internal/server/apis/groups/logo.go]: I'm surprised this doesn't parse this to an int automatically since it's already verifying it's an int as defined :(

File: backend/internal/server/apis/groups/logo.go
Comment (Skylude) [inline backend/internal/server/apis/groups/logo.go:1]: Why is this under "groups" folder? The chat team also wanted to have a "logo" or image for their channels. Do you think we could genericize this to support multiple image types -- maybe on the image table even have a type field as an enum or a separate table? I don't think this decision shoudl block this PR but definitely something we should discuss.

**Review by Skylude — APPROVED** [review]
LGTM besides I'm not sure about the /groups pathing.

File: backend/internal/server/apis/groups/logo.go
Comment (ryanbarlow1) [reply backend/internal/server/apis/groups/logo.go]: Yeah me too

File: backend/internal/server/apis/groups/logo.go
Comment (ryanbarlow1) [reply backend/internal/server/apis/groups/logo.go:1]: @Skylude @tannerwheeler This is currently under groups, but I can adjust this if needed

Comment (tannerwheeler) [reply backend/internal/server/apis/groups/logo.go:1]: @ryanbarlow1 Push it to the groups folder. I can change it with my issue if needed.

File: backend/internal/server/server.go
Comment (tannerwheeler) [inline backend/internal/server/server.go:15]: I believe this folder is added a second time in the imports for the file. Line 20 can be deleted so the imports are in alphabetical order.

File: static/oas/v1/openapi.yml
Comment (tannerwheeler) [inline static/oas/v1/openapi.yml:26]: You will need to change this line to `./groups/season.yml...`

**Review by tannerwheeler — CHANGES_REQUESTED** [review]
@ryanbarlow1 Your Checks are failing because line 1 in the `seasons.go` file `package season` needs to be changed to `package groups`. This file is in /server/apis/groups`.

File: backend/internal/server/apis/groups/logo.go
Comment (ryanbarlow1) [reply backend/internal/server/apis/groups/logo.go:1]: Will do, thanks

File: backend/internal/server/server.go
Comment (ryanbarlow1) [reply backend/internal/server/server.go:15]: I see what you're saying after I merged the latest from main in. Updated.

File: static/oas/v1/openapi.yml
Comment (ryanbarlow1) [reply static/oas/v1/openapi.yml:26]: No the YAML file for season is still under its own folder currently
`static/oas/v1/season/season.yml`

**Review by tannerwheeler — APPROVED** [review]

**Pull request merged**
